variables.tf

```diff
@@ -332,3 +332,14 @@ variable "docker_registry_auth_secret_name" {
   type        = "string"
   default     = ""
 }
+
+variable "default_s3_storage_class" {
```

Comment:
`default_s3_storage_class` --> `s3_storage_class`

Reply:
Okay, I added default as its the default value if user doesnt provide one. I will change it to s3_storage_class 👍

Reply:
Yes so s3_storage_class and no_of_days can be configured when users deploy apiary-datalake by specifying something in variables like this
managed_schemas = [
    {
        schema_name = "dm"
        s3_storage_class = "INTELLIGENT_TIERING"
        s3_lifecycle_policy_transition_period = "50"
    },
    {
        schema_name = "test"
        s3_lifecycle_policy_transition_period = "30"
    },
    {
        schema_name = "ads"
    }
]

If these variables are not provided, then we ll use the default values.

Reply:
We can re-use the block of managed_schemas mentioned above to specify any additional parameters like retention rules and configure it in the bucket policy.

Reply:
Ah, I see. Thanks @spuranda123!